### ISSUE TITLE
[BUG] [pump] [upstream] Center ends recording in Edit Mode

### ISSUE THREAD

**Issue by SHRMP0** [TeamRizu/OutFox#144]
[stepmania/stepmania#97](https://github.com/stepmania/stepmania/issues/97)

According to Kawafumi on Discord: "No comment other than I've checked that it still happens on 4.9.0"

**Comment by SHRMP0** [comment]
Bumping, as requested by @Scraticus

**Comment by JoseVarelaP** [comment]
Yeah, this is gonna be fixed on the editor branch. The reason for this bug is that the Recording mode in the editor for whatever reason is tied to [GAME_BUTTON]s, which depending on what input mode you are (Gameplay / Dedicated buttons), can also act as action buttons (Start/Back/Select),
